config.json

```diff
@@ -66,6 +66,7 @@
     "wordy",
     "allergies",
     "poker",
+    "all-your-base",
```

Comment:
Put new problems last in config list, once they've been solved a few times we can move them to a more appropriate place in the list.
(We probably should document this somewhere.)